### PR TITLE
added documentation changes

### PR DESCRIPTION
**Pull request by bajram-adapt**

### CODE REVIEW

File: docs/WindowAggregation-sparkcompute.md [hunk 1]
```diff
@@ -39,12 +39,41 @@ specifies the value to be added to the value of the current row to get the end b
 
 
 **Aggregates:** Specifies a list of functions to run on the selected window. Supported aggregate functions are `Rank`, 
-`Dense Rank`, `Percent Rank`, `N tile`, `Row Number`, `Median`, `Continuous Percentile`, `Lead`, 
-`Lag`, `First`, `Last`, `Cumulative distribution`, `Accumulate`. Aggregates are specified using syntax: 
+`Dense_Rank`, `Percent_Rank`, `N_Tile`, `Row_Number`, `Median`, `Continuous_Percentile`, `Lead`,
```
Comment (bdmogal) [inline docs/WindowAggregation-sparkcompute.md:42]: is this ALL_CAPS? It seems to be ALL_CAPS in the pipeline JSON.

Comment (ardian4) [reply docs/WindowAggregation-sparkcompute.md:42]: Function names are not case sensitive. 


https://github.com/AdaptiveScale/window-aggregation/blob/d95b30d5d11ace393a1dfe00e5400494f0a6dbab/src/main/java/io/cdap/plugin/WindowAggregationConfig.java#L222-L224

Comment (bdmogal) [reply docs/WindowAggregation-sparkcompute.md:42]: cool.

File: docs/WindowAggregation-sparkcompute.md [hunk 2]
```diff
@@ -39,12 +39,41 @@ specifies the value to be added to the value of the current row to get the end b
...
 
 **Number of partitions:** Number of partitions to use when grouping fields. If not specified, the execution
 framework will decide on the number to use.
+
+
+Sample Pipeline Data
+----------
+Input Records
+
+| Identifier | FirstName | LastName | Value |
```
Comment (bdmogal) [inline docs/WindowAggregation-sparkcompute.md:57]: Please update this after merging #6, since that changes behavior.

Comment (bajram-adapt) [reply docs/WindowAggregation-sparkcompute.md:57]: its in commit 3 for this PR

**Review by CuriousVini — APPROVED** [review]

**Comment by CuriousVini** [comment]
Please squash the commits

**Review by CuriousVini — APPROVED** [review]

**Pull request merged**